test/Android/Breakpoint/TestScript.xml

```diff
@@ -111,7 +111,7 @@
     <Event name="IDebugThreadCreateEvent2" expected="False" />
     <Event name="IDebugOutputStringEvent2" expected="False" />
     <Event name="IDebugBreakpointBoundEvent2" expected="True">
-      <BoundBreakpoint>24,0 to 24,0 in [Unknown]!break_test_1()</BoundBreakpoint>
+      <BoundBreakpoint RegEx="24,0 to 24,0 in (\[Unknown\]|libbreakpoint.so)!break_test_1()"></BoundBreakpoint>
     </Event>
```

Comment:
There is a timing issue here, the breakpoint bound and module load events usually happen very close to each other inside worker thread. Therefore when Glass try to get the breakpoint information from the main thread, MIEngine may or may not finished dealing with the module load event.
This won't cause any issue in product, but in Glass the BoundBreakpoint may or may not have the right module name.